packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-params */
 import {createSelector} from 'reselect';
+import _ from 'lodash';
```

Comment:
We should pull out only the methods we use instead of importing all of lodash.

Reply:
Ugh, we need to go through and do this throughout the code base, I always search and find the `import _`